common/output-model.cpp

```diff
@@ -981,13 +1040,14 @@ void stream_dashboard::draw_dashboard(ux_window& win, rect& r)
                 { pos.x + r.w, pos.y + get_height() }, ImColor(dark_sensor_bg));
 
     auto size = ImGui::CalcTextSize(name.c_str());
-    ImGui::SetCursorPos(ImVec2( r.w / 2 - size.x / 2, 5 ));
+    auto collapse_buton_h = 28 + 3;
```

Comment:
What is 28 and what is 3?
Is it the same 3 used on line 1050 `ImGui::SetCursorPosY( 3.f + collapse_buton_h );`

Reply:
Just add a comment `// Dashboard button size plus some spacing`